bundle/config/mutator/python/python_mutator.go

```diff
@@ -214,6 +230,29 @@ func (m *pythonMutator) runPythonMutator(ctx context.Context, cacheDir string, r
 	return output, pythonDiagnostics
 }
 
+const installExplanation = `If using Python wheels, ensure that 'databricks-pydabs' is included in the dependencies, 
+and that the wheel is installed in the Python environment:
+
+  $ .venv/bin/pip install -e .
+
+If using a virtual environment, ensure it is specified as the venv_path property in databricks.yml, 
+or activate the environment before running CLI commands:
+
+  experimental:
+    pydabs:
+      venv_path: .venv
+`
+
+func explainProcessErr(stderr string) string {
+	if strings.Contains(stderr, "Error while finding module specification for 'databricks.bundles.build'") {
```

Comment:
Is this error message portable across Python versions and OSes?

If so, please include a comment confirming this for future readers.

Reply:
I've checked different implementations for Python. It's very reliable, there are even unit tests using similar regex. The code that produces this message is written in Python and is shared by implementations like pypy.